pkg/yurthub/poolcoordinator/resources/resources.go

```diff
@@ -35,6 +35,7 @@ import (
 )
 
 type PoolScopeResourcesManger struct {
+	//nolint:unused
 	factory                      informers.SharedInformerFactory
```

Comment:
I think the `factory` field here can be directly removed.

Reply:
Ok, I will remove it